src/ui/control/attribution_control.js

```diff
@@ -64,7 +64,6 @@ class AttributionControl {
         this._compactButton.addEventListener('click', this._toggleAttribution);
         this._setElementTitle(this._compactButton, 'ToggleAttribution');
         this._innerContainer = DOM.create('div', 'mapboxgl-ctrl-attrib-inner', this._container);
-        this._innerContainer.setAttribute('role', 'list');
```

Comment:
@kumiko-haraguchi could we just include this change in your PR and revert those other changes? I can make the upstream changes to remove `listitem` without having to intercept them in the dom.

Reply:
@tristen Thank you, I just reverted the commit! https://github.com/mapbox/mapbox-gl-js/pull/12857/commits/73706e6cf47f0b26f0ab166af2e3818c3a56f0f9